File: app/src/main/java/com/faendir/lightning_launcher/multitool/proxy/ImageBitmap.java
package com.faendir.lightning_launcher.multitool.proxy;

import android.graphics.Bitmap;
import android.graphics.Canvas;

/**
 * @author lukas
 * @since 04.07.18
 */
public interface ImageBitmap extends Image {

    Bitmap getBitmap();

    Canvas draw();
}

File: app/src/main/java/com/faendir/lightning_launcher/multitool/proxy/ImageBitmap.kt
package com.faendir.lightning_launcher.multitool.proxy

import android.graphics.Bitmap
import android.graphics.Canvas

/**
 * @author lukas
 * @since 04.07.18
 */
interface ImageBitmap : Image {

    val bitmap: Bitmap

    fun draw(): Canvas
}
